music21/romanText/tsvConverter.py

```diff
@@ -388,30 +420,45 @@ def __init__(self):
         self.dcml_version = 2
 
     @property
-    def beat(self):
+    def beat(self) -> float:
         '''
         'beat' has been removed from DCML v2 in favor of 'mn_onset' and
         'mc_onset'. 'mn_onset' is equivalent to 'beat', except that 'mn_onset'
-        is zero-indexed where 'beat' was 1-indexed. This property reproduces
-        the former 'beat' by adding 1 to 'mn_onset'.
+        is zero-indexed where 'beat' was 1-indexed, and 'mn_onset' is in
+        fractions of a whole-note rather than in quarter notes.
         >>> tabCd = romanText.tsvConverter.TabChordV2()
-        >>> tabCd.mn_onset = 0
+        >>> tabCd.mn_onset = 0.0
+        >>> tabCd.beat
+        1.0
+        >>> tabCd.mn_onset = 1/2
+        >>> tabCd.beat
+        3.0
+        >>> tabCd.beat = 1.5
         >>> tabCd.beat
-        1
+        1.5
         '''
         # beat is zero-indexed in v2 but one-indexed in v1
-        return self.mn_onset + 1
+        # moreover, beat is in fractions of a whole-note in v2
+        return self.mn_onset * 4.0 + 1.0
+
+    @beat.setter
+    def beat(self, beat: float):
+        self.mn_onset = (beat - 1.0) / 4.0 if beat is not None else None
 
     @property
-    def measure(self):
+    def measure(self) -> int:
         '''
         'measure' has been removed from DCML v2 in favor of 'mn' and 'mc'. 'mn'
         is equivalent to 'measure', so this property is provided as an alias.
         '''
         return int(self.mn)
 
+    @measure.setter
+    def measure(self, measure: int):
+        self.mn = int(measure) if measure is not None else None
+
     @property
-    def local_key(self):
+    def local_key(self) -> str:
```

Comment:
these underscore cases are fine because they represent functionality with the same name in DCML v1.